BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/BrainCloudAuthentication.cs

```diff
@@ -86,7 +94,7 @@ public void ClearSavedProfileID()
             object cbObject = null)
         {
             Authenticate(AnonymousId, "", AuthenticationType.Anonymous,
-                              null, forceCreate, success, failure, cbObject);
+                              null, forceCreate, new Dictionary<string, object>(), success, failure, cbObject);
```

Comment:
Replace with `null`. Avoid useless allocation for other calls. If null, don't add it to the server call.